collectors/python.d.plugin/spigotmc/spigotmc.chart.py

```diff
@@ -37,6 +38,19 @@
         ]
     }
 }
+_TPS_REGEX = re.compile(
+    r'^.*: .*?'                 # Message lead-in
+    r'([0-9]{1,2}.[0-9]+), .*?' # 1-minute TPS value
+    r'([0-9]{1,2}.[0-9]+), .*?' # 5-minute TPS value
+    r'([0-9]{1,2}\.[0-9]+).*$', # 15-minute TPS value
+    re.X
+)
+_LIST_REGEX = re.compile(
+    r'^.*?'          # Message lead-in
+    r'([0-9]*)'      # Current user count.
+    r'.*?[0-9]*.*$', # Rest of the line, which should include another number.
+    re.X
```

Comment:
i see from your example the response is

'There are 0/20 players online:'`

But regex looks too complicated for that string. Reasons?

Reply:
Because I meant to add logic to try a different command if we get nothing from `list`, and forgot to do so.

See https://github.com/netdata/netdata/issues/4131#issuecomment-420553899 for the info on the command and what it's output format is.

Pretty much, with this regex, we can successfully match the original `list` command's output, or the output from the `online` command mentioned in that comment with the same regex, and in both cases we don't have to care as much about surrounding formatting.

Reply:
```python
In [8]: _LIST_REGEX = re.compile( 
      :     r'^.*?'          # Message lead-in 
      :     r'([0-9]*)'      # Current user count. 
      :     r'.*?[0-9]*.*$', # Rest of the line, which should include another num
      : ber. 
      :     re.X)                                                                

In [9]: _LIST_REGEX.match('There are 0/20 players online:').groups()             
Out[9]: ('',)

```